SIR-X.ipynb

```diff
@@ -240,6 +243,268 @@
oFjcebv5KG80tFLd0JZ4NSZ+tnZGj7Urys/m3FmTuPl98xM7/9mTOH1ivv76l5SlQBAZQEtnD7VNR6lp7qC2+Sg1TYmftc0dvHmwna5o/FjbqRPyKJs2gWveM5OF0yawcNoEyqZNoKQwTzt/SSsKBBlz2ruiNLR2caClk4bWLhpaOtl/pPMdO/+WPn/pAxTkZjG7eDyzJo/j0oVTKZue2PEvLClk4njdCSyZQYEgac/dae2K0tzeTVN7N80d3TS199DU3kVja1e/nX8XbV3Rd20jPyfCrMmJHf4FcyYza/K4YwEwe/J4Jo3P0V/7kvGSCgQzWwn8XyAL+L67/2O/9/OAHwFLgUPAp919b/De14CbgBjwRXd/Kpltytji7hztidFyNEpLZw8tR3to6eyhtTMaTEePLTtytIfm9h6a2rtp6uimub2baO8lO/3kZUeYVpTH9MJ8zjqtiMvK8phelM+0wsTP6UV5TCvMp2hctnb4MuYNGQhmlgXcDVwJ1AKbzKzC3Xf0aXYT0OzuC81sNXAn8GkzWwysBs4GTgeeMbNFwTpDbVNSiLvTE3O6Y3G6emJ0RuMc7Y7R2ZN4He2JcbQ78TOxLM7Rnhgd3TE6uqK0d8do74rS0R2lrStKRzDf3hWjvTsxHxtkp94rLztC0bgcivKzKS7IZe6U8Zw/ZxKTC3IpHp9LcUHi1Ts/qSCHwjzt6EWSlcwIYRlQ7e57AMzsEWAV0HfnvQq4PZh+FLjLEv8KVwGPuHsX8KaZVQfbI4ltphV3xx3i7sT6TseduCe+iiDmfuxnLO7E4/zXdPDz2CuYj8YS70XjTiwef8d8NOb0xOLBdPy/lgXt+i7vjsbpjsXpicbpicWP7dx7el9RpysWpzsapysao6snfmzn3x2L0xWN48ffXw9qfG4W43OzmZDX+zOxQ59dPJ6C3CwK8rIpyM2mMD+bwvwcisZlU5Sfc2znX5ifQ2F+Nvk5WcP7oYnIOyQTCDOBmj7ztcDywdq4e9TMjgBTguUv9Vt3ZjA91DaHzc0PbOLNg+044B7svEnssBPz71wW77Osdz7uDs475vsHQCrKjhg5WRFysozc7EgwHTk2nZtlx5ZNzM0hLzvxXt6xV1a/ZVnkZkfIz4mQn5PFuJwsxuVmHZvOD+YT0xHys7OIRPQXukg6SCYQBvrX3H/3N1ibwZYP9LimAXepZnYLcAvAnDlzBq/yOOZOKSAvOwszMDMMiPROGxiJn5FgOhJJvBcxiJgRCQ45RHqXRYJtRN7Zpu/7kT7rm0FWxMgKlmdFjCwzIhEjK8I7lvW2y4oY2ZEIkQhkRyLBfP/3Ezvz7KxE2+yIkR3s4Hvb6nCJiCQrmUCoBWb3mZ8F1A3SptbMsoGJQNMQ6w61TQDc/V7gXoDy8vKT+jv8Gx9dfDKriYiMKck8WHUTUGZmpWaWS+IkcUW/NhXAjcH0tcCz7u7B8tVmlmdmpUAZ8HKS2xQRkVE05AghOCdwK/AUiUtE73f37WZ2B1Dp7hXAfcCDwUnjJhI7eIJ2a0icLI4Cf+ruMYCBtjn83RMRkWSZp+rZ0AGUl5d7ZWVl2GWIiKQVM9vs7uVDtUvmkJGIiIwBCgQREQEUCCIiElAgiIgIoEAQEZFAWl1lZGaNwL6TXH0qcHAYywlbpvUHMq9PmdYfyLw+ZVp/YOA+zXX3kqFWTKtAOBVmVpnMZVfpItP6A5nXp0zrD2RenzKtP3BqfdIhIxERARQIIiISGEuBcG/YBQyzTOsPZF6fMq0/kHl9yrT+wCn0acycQxARkeMbSyMEERE5jowPBDNbaWY7zazazL4adj3Dwcz2mlmVmW0xs7T8tj8zu9/MGsxsW59lxWb2tJm9EfycHGaNJ2KQ/txuZm8Hn9MWM/tImDWeCDObbWbPmdlrZrbdzP48WJ7On9FgfUrLz8nM8s3sZTN7NejPt4LlpWa2MfiMfho8YiC5bWbyISMzywJ2AVeSeFjPJuB6d0/bZzdDIhCAcndP2+unzewyoA34kbsvCZb9E9Dk7v8YhPdkd/+fYdaZrEH6czvQ5u7fDrO2k2FmM4AZ7v6KmRUCm4FrgD8mfT+jwfr0KdLwcwqeW1/g7m1mlgP8Fvhz4MvAz939ETP7N+BVd78nmW1m+ghhGVDt7nvcvRt4BFgVck0CuPvzJJ6d0dcq4IFg+gES/1jTwiD9SVvuXu/urwTTrcBrJJ6Hns6f0WB9Skue0BbM5gQvBz4APBosP6HPKNMDYSZQ02e+ljT+H6APB/7DzDYHz5zOFNPdvR4S/3iBaSHXMxxuNbOtwSGltDm80peZzQPOBzaSIZ9Rvz5Bmn5OZpZlZluABuBpYDdw2N2jQZMT2udleiAM9IT5TDhGdom7XwB8GPjT4HCFpJ57gAXAe4B64DvhlnPizGwC8BjwJXdvCbue4TBAn9L2c3L3mLu/h8Rz6ZcBZw3ULNntZXog1AKz+8zPAupCqmXYuHtd8LMBeJzE/wiZ4EBwnLf3eG9DyPWcEnc/EPyDjQP/Tpp9TsFx6ceAH7v7z4PFaf0ZDdSndP+cANz9MLAeuAiYZGa9j0c+oX1epgfCJqAsOOueS+JZzxUh13RKzKwgOCGGmRUAHwK2HX+ttFEB3BhM3wg8EWItp6x3xxn4OGn0OQUnLO8DXnP3/9PnrbT9jAbrU7p+TmZWYmaTgulxwAdJnBd5Drg2aHZCn1FGX2UEEFxC9i9AFnC/u/+vkEs6JWY2n8SoACAbeDgd+2RmPwFWkPhmxgPAN4FfAGuAOcBbwHXunhYnagfpzwoShyEc2AulhJHYAAAAdklEQVR8rvf4e6ozs0uB3wBVQDxY/Nckjrmn62c0WJ+uJw0/JzM7l8RJ4ywSf9yvcfc7gn3EI0Ax8HvgM+7eldQ2Mz0QREQkOZl+yEhERJKkQBAREUCBICIiAQWCiIgACgQREQkoEEREBFAgiIhIQIEgIiIA/H/pKn0pHHqzxAAAAABJRU5ErkJggg==\n",
```

Comment:
Is this an image generated by the notebook?

Reply:
Not sure if we should be version controlling this. I am not familiar with this haha, but I know that base64 images are heavy. Should we also version control the results of executing the notebook?

Reply:
Output removed!